src/main/java/org/damap/base/conversion/AbstractTemplateExportScienceEuropeComponents.java

```diff
@@ -363,11 +391,26 @@ public void datasetsInformation() {
 
   /** storageIntroInformation */
   public void storageIntroInformation() {
-    String coordinatorFullName;
+    String coordinatorFullName = null;
```

Comment:
I have queried Christiane and Tomasz for the behaviour we want in this part. 
 The first contributor choice should be the data manager role. So we need to query all contributors and not only the coordinators.
 Second choice is the project leader, then prinvcipal investigator then project coordinator. 
 If none of these roles exist, the fallback will be the contact person.
 
As a last resort, you should choose the string `"the project leader`.